src/compas_timber/connections/l_miter.py

```diff
@@ -4,7 +4,7 @@
 from compas.geometry import Vector
 from compas.geometry import cross_vectors
 
-from compas_timber.elements.features import CutFeature
+from compas_timber._fabrication import JackRafterCut
```

Comment:
_fabrication -> fabrication?

Reply:
starting with this PR, there are two parallel fabrication related systems (as @papachap put it nicely, the old one is joint-centric and the new one is feature/process-centric). This is an attempt to let them live side-by-side until all feature types are implemented in the new system. 

Eventually, `_fabrication` will get renamed and replace the existing `fabrication`.